docs/project/governance.md

```diff
@@ -0,0 +1,97 @@
+the continued development of the project through ongoing engagement with the
+community. They have shown they can be trusted to maintain pyodide with
+care. Being a core developer allows contributors to more easily carry on
+with their project related activities by giving them direct access to the
+project’s repository and is represented as being a member of the core team on the
+pyodide [GitHub organization](https://github.com/orgs/pyodide/teams/core/members).
+Core developers are expected to review code
+contributions, can merge approved pull requests, can cast votes for and against
+merging a pull-request, and can be involved in deciding major changes to the
+API.
+
+New core developers can be nominated by any existing core developers. Once they
+have been nominated, there will be a vote by the current core developers.
+Voting on new core developers is one of the few activities that takes place on
+the project's private communication channels. While it is expected that most votes
+will be unanimous, a two-thirds majority of the cast votes is enough. The vote
+needs to be open for at least 1 week.
+
+Core developers that have not contributed to the project (commits or GitHub
+comments) in the past 2 years will be asked if they want to become emeritus
+core developers and recant their commit and voting rights until they become
+active again.
+
+
+## Decision Making Process
+
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project contributors' [issue
+tracker](https://github.com/pyodide/pyodide/issues) and on [Github
+discussion](https://github.com/pyodide/pyodide/discussions).
+Occasionally, sensitive discussion occurs on a private communication channels.
+
+pyodide uses a "consensus seeking" process for making decisions. The group
+tries to find a resolution that has no open objections among core developers.
+At any point during the discussion, any core-developer can call for a vote,
+which will conclude two weeks from the call for the vote. This is what we
+hereafter may refer to as “the decision making process”.
+
+Decisions (in addition to adding core developers as above)
+are made according to the following rules:
+
+* **Minor Documentation changes and minor build setup changes**, include for
+  instance improving the wording in the documentation, updating CI or
+  dependencies.  Core developers are expected to give “reasonable time” to
+  others to give their opinion on the pull request if they’re not confident
+  others would agree. If no review is received within this time, the Pull
+  Request can be merged.
+
+* **Code changes impacting user facing APIs, or having backward compatibility
+  implications** require a review and an approval (+1 vote) by a core
+  developer, no objections with -1 vote by a core developer (lazy consensus).
+  This process happens on the pull-request page.
```

Comment:
```suggestion
  implications** require review and approval by a core
  developer and no objections raised by any core developer (lazy consensus).
  This process happens on the pull-request page.
```
Maybe say again that there should be reasonable time given and also suggest @notifying community members that one expects to have a stake in the matter? Maybe also clarify that it needs to be reviewed by a core developer different from the one who opened the PR?

I think it's confusing to say +1 vote / -1 vote because it's not clear to me that there is any formal way to do that / any formal difference between "I object to this" and "-1"?

Reply:
> suggest notifying community members that one expects to have a stake in the matter?

I think it's up to the contributor to decide about mentioning or not. Generally maintainers are often following the repo and will see the PR happen. Whether they will respond or not depends on their availability, interest in the particular topic etc and so systematically mentioning is nor really useful if you follow all issues anyway.

> Maybe also clarify that it needs to be reviewed by a core developer different from the one who opened the PR?

You need an approval to merge those PR and you can't approve your own PRs in the Github UI so I think that excludes this case.

Reply:
> you can't approve your own PRs in the Github UI

Oh okay, I hadn't tried that yet.